test/multiverse/suites/tilt/tilt_instrumentation_test.rb

```diff
@@ -6,13 +6,184 @@ class TiltInstrumentationTest < Minitest::Test
   include MultiverseHelpers
```

Comment:
I wonder if we're actually using any of these in this PR... we might be able to pull this out.

Reply:
The multiverse helpers? hmmm. I'm not very familiar with that part of multiverse, but looking at it it seems to have some things that are used indirectly. I _think_ we do need to keep that since I believe it's needed for multiverse to do all of the set up it needs to for the agent and fake collector? We could experiment with that though.

Reply:
I'll remove them for this CI run and if trouble brews, I'll stick them back in.

Reply:
CI passed! Marking this as resolved.

Reply:
Huh, that's surprising! I wonder how those things are happening then? Multiverse has some mystery to it lol. thanks for checking that out!